apps/commudle-admin/src/app/feature-modules/hackathon-control-panel/components/hackathon-control-panel-tracks-prizes/hackathon-control-panel-tracks-prizes.component.ts

```diff
@@ -5,6 +5,9 @@ import { NbRouteTab } from '@commudle/theme';
 import { faArrowRight, faGamepad, faMicrophone, faRectangleList } from '@fortawesome/free-solid-svg-icons';
 import { HackathonService } from 'apps/commudle-admin/src/app/services/hackathon.service';
 import { Subscription } from 'rxjs';
+import { ICommunity } from 'apps/shared-models/community.model';
+import { ICommunityGroup } from 'apps/shared-models/community-group.model';
```

Comment:
let's use @commmudle/shared-models for ICommunity and SeoService from @commudle/shared-service